homeassistant/components/device_tracker/__init__.py

```diff
@@ -89,7 +90,7 @@ def is_on(hass: HomeAssistantType, entity_id: str=None):
 def see(hass: HomeAssistantType, mac: str=None, dev_id: str=None,
         host_name: str=None, location_name: str=None,
         gps: GPSType=None, gps_accuracy=None,
-        battery=None):  # pylint: disable=too-many-arguments
+        battery=None, fuel_level=None):  # pylint: disable=too-many-arguments
```

Comment:
I feel the refactor should be a separate PR - as that will no longer make this an atomic PR. But I do like the idea of changing it to a dictionary of properties